src/AdvApi32.Desktop/AdvApi32.Helpers.cs

```diff
@@ -246,8 +246,9 @@ public static unsafe void CreateService(string lpBinaryPathName, string lpServic
 
                     fixed (void* lpInfo = new byte[Marshal.SizeOf(descriptionStruct)])
                     {
-                        Marshal.StructureToPtr(descriptionStruct, new IntPtr(lpInfo), false);
-                        if (!ChangeServiceConfig2(svcHandle, ServiceInfoLevel.SERVICE_CONFIG_DESCRIPTION, lpInfo))
+                        var ptr = new IntPtr(lpInfo);
+                        Marshal.StructureToPtr(descriptionStruct, ptr, false);
+                        if (!ChangeServiceConfig2(svcHandle, ServiceInfoLevel.SERVICE_CONFIG_DESCRIPTION, ptr))
```

Comment:
I see you also are calling the `IntPtr` overload rather than the regular pointer overload. Between that and all the xml doc comment overload changes you made, it makes me wonder if something happened to remove the pointer overload of the method. Is that what happened? That would be concerning.